dev-lang/janet/janet-1.6.0.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Gentoo Authors
```

Comment:
This ebuild seems useless as you have a newer one available.

Reply:
I removed the ebuild for 1.6.0